.github/workflows/tag-and-publish.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Fetch Jira ID from commit message
         id: ensure-jira-id
         run: |
-          JIRA_ID=$(echo '${{ github.event.head_commit.message }}' | grep -Eo '\[?[A-Z][A-Z]+-[0-9]+\]?')
+          JIRA_ID=$(echo '${{ github.event.head_commit.message }}' | grep -Eo '[A-Z][A-Z]+-[0-9]+')
```

Comment:
Just to clarify, this will work fine for commits with Jira ID without the `[]` or `()` no?